### ISSUE TITLE
Drift migrator not updating part directive in DAOs

### ISSUE THREAD

**Issue by kuhnroyal** [simolus3/moor#1558]
I just migrated to drift and everything worked pretty well.
The only thing I noticed was that my DAO classes still had part directives with `moor` suffix while the actually files got renamed.

**Comment by simolus3** [comment]
Glad to hear that it worked well for the most part!

> The only thing I noticed was that my DAO classes still had part directives with `moor` suffix while the actually files got renamed.

I'm not entirely sure what you mean with that, `include`d moor files from a `@UseDao` annotation? I thought I had handled that, and I've added [a test](https://github.com/simolus3/moor/blob/39b4745b7d312063dcb6233b42dd8e8167259212/drift_dev/test/cli/migrate_test.dart#L178-L204) too. Has that not been working for you or are you referring to another thing?
